zerver/webhooks/stripe/view.py

```diff
@@ -78,9 +78,8 @@ def api_stripe_webhook(request, user_profile, client,
                     body = body_template.format(id=object_id, link=link)
 
                 else:
-                    end_time = datetime.fromtimestamp(data_object["trial_end"]).strftime('%b %d %Y at %I:%M%p')
-                    body_template = "The customer subscription trial with id **[{id}]({link})** will end on {time}"
-                    body = body_template.format(id=object_id, link=link, time=end_time)
+                    body_template = "The customer subscription trial with id **[{id}]({link})** will end in three days."
```

Comment:
Basing it off of 
![image](https://cloud.githubusercontent.com/assets/890911/23500523/4a3e5226-fee5-11e6-90c9-af106481d1c0.png)
at https://stripe.com/docs/api/python#event_types